drivers/hopwa_caper/app/models/hopwa_caper/generators/fy2024/generator.rb

```diff
@@ -0,0 +1,186 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# run manually with
+# Reporting::Hud::RunReportJob.new.perform(HopwaCaper::Generators::Fy2024::Generator.name, HudReports::ReportInstance.last, email:false)
+module HopwaCaper::Generators::Fy2024
+  class Generator < ::HudReports::GeneratorBase
+    def self.fiscal_year
+      'FY 2024'
+    end
+
+    def self.generic_title
+      'HOPWA CAPER'
+    end
+
+    def self.short_name
+      'HOPWA CAPER'
+    end
+
+    def self.file_prefix
+      "v1.0 #{short_name} #{fiscal_year}"
+    end
+
+    def self.default_project_type_codes
+      HudUtility2024.path_project_type_codes
```

Comment:
Pretty sure this should be some other set of project types, maybe PH?

Reply:
Or maybe
 ```
HudUtility2024.spm_project_type_codes
```

Reply:
oh thanks, I think that's left-over from when I copied this from the path report. I don't think the method is called though, I'll just take it out

Reply:
That gets used if you have never run a HOPWA CAPER to set the default project types that should be included.  It was also limiting which project types were being included, but that might have been an odd side effect.  You can try removing it, but if you do, we'll need to test the scenario where no HOPWA CAPERs have been run before.